web/js/date/wv.date.timeline.zoom.js

```diff
@@ -1,18 +1,11 @@
-/**
- * @module wv.date.timeline
- */
-var wv = wv || {};
-wv.date = wv.date || {};
-wv.date.timeline = wv.date.timeline || {};
+import $ from 'jquery';
+import uiMouse from '../ui/wv.ui.mouse';
```

Comment:
This is slightly different in Jo's branch. He is exporting this from the parent ui component and the `wv.` has been dropped: https://github.com/nasa-gibs/worldview/tree/modularize-ui/web/js/ui